src/modules/Plans/views/details/tabs/VirtualMachines/Migration/MigrationVirtualMachinesList.tsx

```diff
@@ -246,7 +246,7 @@ export const MigrationVirtualMachinesList: FC<{ planData: PlanData }> = ({ planD
     plan?.status?.migration?.vms || [];
 
   const vmDict: Record<string, V1beta1PlanStatusMigrationVms> = {};
-  migrationVirtualMachines.forEach((migration) => (vmDict[migration.id] = m));
+  migrationVirtualMachines.forEach((migration) => (vmDict[migration.id] = migration));
```

Comment:
how did it ever worked before? who is m? 
BTE this file should also be heavily refactored

Reply:
Looks like [these snuck in](https://github.com/kubev2v/forklift-console-plugin/commit/a01c3bcbb0d38dfe8444cb1b79a8f636542631e6#diff-32eeb65a7dad86d67b4477e1569b66beb098eaf05df0980701874ce71b14a3a6L247) recently and were not caught in review, oops!

Reply:
I know the guy that did it :)

Reply:
Fixed them all here
https://github.com/kubev2v/forklift-console-plugin/pull/1563